tour_model_eval/get_users.py

```diff
@@ -0,0 +1,31 @@
+import data_preprocessing as preprocess
+
+
+# to determine if the user is valid:
+# valid user should have >= 10 trips for further analysis and the proportion of filter_trips is >=50%
+def valid_user(filter_trips,trips):
+    valid = False
+    if len(filter_trips) >= 10 and len(filter_trips) / len(trips) >= 0.5:
+        valid = True
+    return valid
+
+
+# - user_ls: a list of all users
+# - valid_user_ls: a list of valid users
+def get_user_ls(all_users,radius):
```

Comment:
how is `user_ls` different from `all_users`?

Reply:
`user_ls` is like user1, user1
`all_users` is user id

Reply:
Add a comment here for others to read as well